### PR TITLE
update the the dahsboard(zh) version

### PR DESCRIPTION
**Pull request by zehuaiWANG**
This PR is used to update the Chinese version of dashboard and add myself to zh/OWNERS. I am familiar with both dashboard and Chinese, and I am willing to do it.
/assign @maciaszczykm
/assign @floreks 
/assign @hwdef
/assign @shu-mutou
Thanks.

### CODE REVIEW

File: i18n/zh/messages.zh.xlf [hunk 1]
```diff
@@ -71,11 +71,11 @@
...
+    <x id="START_TAG_SPAN" ctype="x-span" equiv-text="&lt;span>"/> 确认删除 <x id="INTERPOLATION" equiv-text="{{data.displayName}}"/> <x id="START_ITALIC_TEXT" ctype="x-i" equiv-text="&lt;i>"/><x id="INTERPOLATION_1" equiv-text="{{data.objectMeta.name}}"/><x id="CLOSE_ITALIC_TEXT" ctype="x-i" equiv-text="&lt;/i>"/><x id="CLOSE_TAG_SPAN" ctype="x-span" equiv-text="&lt;/span>"/>
     <x id="START_TAG_SPAN_1" ctype="x-span" equiv-text="&lt;span>"/>
-       in namespace <x id="START_ITALIC_TEXT" ctype="x-i" equiv-text="&lt;i>"/><x id="INTERPOLATION_2" equiv-text="{{data.objectMeta.namespace}}"/><x id="CLOSE_ITALIC_TEXT" ctype="x-i" equiv-text="&lt;/i>"/>
+       在命名空间中 <x id="START_ITALIC_TEXT" ctype="x-i" equiv-text="&lt;i>"/><x id="INTERPOLATION_2" equiv-text="{{data.objectMeta.namespace}}"/><x id="CLOSE_ITALIC_TEXT" ctype="x-i" equiv-text="&lt;/i>"/>
     <x id="CLOSE_TAG_SPAN" ctype="x-span" equiv-text="&lt;/span>"/>
-    <x id="START_TAG_SPAN" ctype="x-span" equiv-text="&lt;span>"/>?<x id="CLOSE_TAG_SPAN" ctype="x-span" equiv-text="&lt;/span>"/>
+    <x id="START_TAG_SPAN" ctype="x-span" equiv-text="&lt;span>"/>吗?<x id="CLOSE_TAG_SPAN" ctype="x-span" equiv-text="&lt;/span>"/>
```
Comment (hwdef) [inline i18n/zh/messages.zh.xlf]: Is this change necessary?

File: i18n/zh/messages.zh.xlf [hunk 2]
```diff
@@ -1264,7 +1264,7 @@
       </trans-unit>
       <trans-unit id="dc6f44635b83446c0f7f31ad734c33c6af1f7e7a" datatype="html">
         <source>Config Maps</source>
-        <target state="new">配置图</target>
+        <target state="new">Config Maps</target>
```
Comment (hwdef) [inline i18n/zh/messages.zh.xlf]: why change this from Chinese to English?

File: i18n/zh/messages.zh.xlf [hunk 3]
```diff
@@ -1280,7 +1280,7 @@
       </trans-unit>
       <trans-unit id="473fb06a7b7b3992af493059e8f9c8c74f369a97" datatype="html">
         <source>Dependencies</source>
-        <target state="new">Dependencies</target>
+        <target state="new">依赖：</target>
```
Comment (hwdef) [inline i18n/zh/messages.zh.xlf]: Original has no ":"

File: i18n/zh/messages.zh.xlf [hunk 4]
```diff
@@ -1958,7 +1958,7 @@
       </trans-unit>
       <trans-unit id="016ff9645a730c7a1171f2df1858a8f74d8e9ec0" datatype="html">
         <source>UID</source>
-        <target state="new">UID</target>
+        <target state="new">唯一ID</target>
```
Comment (hwdef) [inline i18n/zh/messages.zh.xlf]: Is this change necessary?

File: i18n/zh/messages.zh.xlf [hunk 5]
```diff
@@ -2162,15 +2162,15 @@
       </trans-unit>
       <trans-unit id="b0e6fb912650bbbe88ff81601b2dea87727abb6f" datatype="html">
         <source>Persistent Volume Claims</source>
-        <target state="new">持久化 Volume 请求</target>
+        <target state="new">持久化卷请求</target>
```
Comment (hwdef) [inline i18n/zh/messages.zh.xlf]: Is this change necessary?

File: i18n/zh/messages.zh.xlf [hunk 6]
```diff
@@ -2162,15 +2162,15 @@
...
         <context-group purpose="location">
           <context context-type="sourcefile">../src/app/frontend/common/components/resourcelist/persistentvolumeclaim/template.html</context>
           <context context-type="linenumber">21</context>
         </context-group>
       </trans-unit>
       <trans-unit id="796e42bec8a6996607cf1f3e80235b86d695804a" datatype="html">
         <source>Volume</source>
-        <target state="new">Volume</target>
+        <target state="new">卷</target>
```
Comment (hwdef) [inline i18n/zh/messages.zh.xlf]: [Terms](https://github.com/k8smeetup/k8s-official-translation/blob/master/trans-glossary.md) should not be translated

File: i18n/zh/messages.zh.xlf [hunk 7]
```diff
@@ -2465,7 +2465,7 @@
       <trans-unit id="d094e262a8d26c66b5eb9f7dbf501cd711c1306f" datatype="html">
         <source>Daemon Sets
       </source>
-        <target state="new">Daemon 设置
+        <target state="new">守护进程集
```
Comment (hwdef) [inline i18n/zh/messages.zh.xlf]: same as above.

**Comment by hwdef** [comment]
Many of your translations are unnecessary.May cause people confusion.
And I noticed that you want to become an approver.It would be better if you could provide some previous translation works.

File: i18n/zh/messages.zh.xlf (hunk 2 above)
Comment (zehuaiWANG) [reply i18n/zh/messages.zh.xlf]: Sorry I make a mistake， it will be better to translate it as “配置字典”.

File: i18n/zh/messages.zh.xlf (hunk 7 above)
Comment (zehuaiWANG) [reply i18n/zh/messages.zh.xlf]: sets is translated as “集合” not “设置” in chinese

Comment (zehuaiWANG) [reply i18n/zh/messages.zh.xlf]: It is obvious that Daemon Sets is translated into Daemon settings is wrong.

**Comment by zehuaiWANG** [comment]
I don't think it's unnecessary. It's also make people confused to include English in the translation. And some of the translation is obviously wrong. Do you think they are also unnecessary?

**Comment by zehuaiWANG** [comment]
Turn off PR because I don't want to argue this with you. It‘s unnecessary.

**Comment by zehuaiWANG** [comment]
Such a translation can only show that you are not familiar with the resource type of kubernetes. You do not even know that daemonset is a resource type rather than a setting in kubernetes.

**Comment by zehuaiWANG** [comment]
Such a translation can only show that you are not familiar with the resource type of kubernetes. You do not even know that daemonset is a resource type rather than a setting of kubernetes.

**Comment by hwdef** [comment]
@zehuaiWANG 
please check this [repo](https://github.com/k8smeetup/k8s-official-translation).
you can find translation specification.

**Comment by hwdef** [comment]
And if you want to be an approver,you should become a memeber of kubernetes org before.

File: i18n/zh/messages.zh.xlf (hunk 7 above)
Comment (hwdef) [reply i18n/zh/messages.zh.xlf]: I think this place is should keep "Daemon sets", do not translate.

**Comment by hwdef** [comment]
I am not targeting you.
We can have a friendly conversation.

**Comment by zehuaiWANG** [comment]
Please give me your WeChat Id or any other contact information(such as email).  I am not targeting you also. In my opinion, some of the translation is obviously wrong, It's just easier to modify it to be a approver.

File: i18n/zh/messages.zh.xlf (hunk 7 above)
Comment (zehuaiWANG) [reply i18n/zh/messages.zh.xlf]: Of course, you can chose to not translate all kubernetes resource types at all, but you shouldn't just translate some of them and tell me that is unnecessary to tranlate the other. Half of translation which makes the whole page look very strange.

**Comment by hwdef** [comment]
I already send a request.

File: i18n/zh/messages.zh.xlf [hunk 8]
```diff
@@ -71,11 +71,11 @@
     <x id="START_TAG_SPAN" ctype="x-span" equiv-text="&lt;span>"/>?<x id="CLOSE_TAG_SPAN" ctype="x-span" equiv-text="&lt;/span>"/>
   </source>
         <target state="new">
-    <x id="START_TAG_SPAN" ctype="x-span" equiv-text="&lt;span>"/>Are you sure you want to delete <x id="INTERPOLATION" equiv-text="{{data.displayName}}"/> <x id="START_ITALIC_TEXT" ctype="x-i" equiv-text="&lt;i>"/><x id="INTERPOLATION_1" equiv-text="{{data.objectMeta.name}}"/><x id="CLOSE_ITALIC_TEXT" ctype="x-i" equiv-text="&lt;/i>"/><x id="CLOSE_TAG_SPAN" ctype="x-span" equiv-text="&lt;/span>"/>
+    <x id="START_TAG_SPAN" ctype="x-span" equiv-text="&lt;span>"/> 确认删除 <x id="INTERPOLATION" equiv-text="{{data.displayName}}"/> <x id="START_ITALIC_TEXT" ctype="x-i" equiv-text="&lt;i>"/><x id="INTERPOLATION_1" equiv-text="{{data.objectMeta.name}}"/><x id="CLOSE_ITALIC_TEXT" ctype="x-i" equiv-text="&lt;/i>"/><x id="CLOSE_TAG_SPAN" ctype="x-span" equiv-text="&lt;/span>"/>
```
Comment (hwdef) [inline i18n/zh/messages.zh.xlf]: 多了个空格

File: i18n/zh/messages.zh.xlf [hunk 9]
```diff
@@ -1826,31 +1826,31 @@
       </trans-unit>
       <trans-unit id="90ae2a4959c979abc049f4652f59b85e993e16cd" datatype="html">
         <source>CPU requests (cores)</source>
-        <target state="new">CPU requests (cores)</target>
+        <target state="new">CPU使用(核)</target>
```
Comment (hwdef) [inline i18n/zh/messages.zh.xlf]: 中英文空格

File: i18n/zh/messages.zh.xlf [hunk 10]
```diff
@@ -1826,31 +1826,31 @@
...
         <context-group purpose="location">
           <context context-type="sourcefile">../src/app/frontend/common/components/resourcelist/node/template.html</context>
           <context context-type="linenumber">79</context>
         </context-group>
       </trans-unit>
       <trans-unit id="d9465e327fed60c8566555ae63a51d8d24b4dcbe" datatype="html">
         <source>CPU limits (cores)</source>
-        <target state="new">CPU limits (cores)</target>
+        <target state="new">CPU配额 (核)</target>
```
Comment (hwdef) [inline i18n/zh/messages.zh.xlf]: 中英文空格

File: i18n/zh/messages.zh.xlf (hunk 4 above)
Comment (hwdef) [inline i18n/zh/messages.zh.xlf]: 这里我觉得还是UID好一点。

File: i18n/zh/messages.zh.xlf [hunk 11]
```diff
@@ -2046,23 +2046,23 @@
...
+        <target state="new">期望值</target>
         <context-group purpose="location">
           <context context-type="sourcefile">../src/app/frontend/common/components/podstatus/template.html</context>
           <context context-type="linenumber">87</context>
         </context-group>
       </trans-unit>
       <trans-unit id="f2db0be1c93c93d5af51ed6d3b42abb184d07c08" datatype="html">
         <source>CPU Usage (cores)</source>
-        <target state="new">CPU使用率 (cores)</target>
+        <target state="new">CPU使用率 (核)</target>
```
Comment (hwdef) [inline i18n/zh/messages.zh.xlf]: 中英文空格

File: i18n/zh/messages.zh.xlf [hunk 12]
```diff
@@ -2688,23 +2688,23 @@
...
+        <target state="new">使用认证头登录</target>
         <context-group purpose="location">
           <context context-type="sourcefile">../src/app/frontend/chrome/userpanel/template.html</context>
           <context context-type="linenumber">22</context>
         </context-group>
       </trans-unit>
       <trans-unit id="2026658d9603d4eea6cbf142234d657111385bbc" datatype="html">
         <source>Logged in with token</source>
-        <target state="new">使用 token 登录</target>
+        <target state="new">使用token登录</target>
```
Comment (hwdef) [inline i18n/zh/messages.zh.xlf]: 中英文空格。

File: i18n/zh/messages.zh.xlf [hunk 13]
```diff
@@ -2918,7 +2918,7 @@
       </trans-unit>
       <trans-unit id="860f8d6a2a2265b9bdf4e5b35401ae1069082fe7" datatype="html">
         <source>General-purpose web UI for Kubernetes clusters</source>
-        <target state="new">Kubernetes集群的通用Web UI</target>
+        <target state="new">Kubernetes集群的通用Web界面</target>
```
Comment (hwdef) [inline i18n/zh/messages.zh.xlf]: 中英文空格

File: i18n/zh/messages.zh.xlf [hunk 14]
```diff
@@ -3239,7 +3239,7 @@
               Data must be Base64 encoded.
             </source>
         <target state="new">
-              Data 必须是 Base64 编码的。
+              数据必须是Base64编码的。
```
Comment (hwdef) [inline i18n/zh/messages.zh.xlf]: 中英文空格。

File: i18n/zh/messages.zh.xlf [hunk 15]
```diff
@@ -3248,7 +3248,7 @@
       </trans-unit>
       <trans-unit id="f0ca722171e7547ce4fd039386dc5fc35359e186" datatype="html">
         <source>Specify the data for your secret to hold. The value is the Base64 encoded content of a .dockercfg file.</source>
-        <target state="new">指定要保留的 secret 的 data 。该值是 .dockercfg 文件中的 Base64 编码内容。</target>
+        <target state="new">指定要保留的密钥的数据。该值是.dockercfg文件中的Base64编码内容。</target>
```
Comment (hwdef) [inline i18n/zh/messages.zh.xlf]: 中英文空格

File: i18n/zh/messages.zh.xlf [hunk 16]
```diff
@@ -3366,15 +3366,15 @@
       </trans-unit>
       <trans-unit id="43f286635565c58e36280035a459dc47dfd62868" datatype="html">
         <source>Enter the URL of a public image on any registry, or a private image hosted on Docker Hub or Google Container Registry.</source>
-        <target state="new">输入公共镜像的 URL （可以是在私有仓库或Docker Hub或Google Container Registry上托管任何镜像URL）。</target>
+        <target state="new">输入公共镜像的连接（可以是在私有仓库或Docker Hub或Google Container Registry上托管任何镜像连接）。</target>
```
Comment (hwdef) [inline i18n/zh/messages.zh.xlf]: 中英文空格

File: i18n/zh/messages.zh.xlf [hunk 17]
```diff
@@ -3409,7 +3409,7 @@
         Setting high number of pods may cause performance issues of the cluster and Dashboard UI.
       </source>
         <target state="new">
-        设置大量 pod 可能会导致集群和 Dashboard UI 出现性能问题。
+        设置大量容器组可能会导致集群和Dashboard界面出现性能问题。
```
Comment (hwdef) [inline i18n/zh/messages.zh.xlf]: 中英文空格。

File: i18n/zh/messages.zh.xlf [hunk 18]
```diff
@@ -3532,23 +3532,23 @@
...
+        <target state="new">如果指定的镜像是私有的，则可能需要pull secret credential。 您可以选择现有密钥或创建新密钥。</target>
         <context-group purpose="location">
           <context context-type="sourcefile">../src/app/frontend/create/from/form/template.html</context>
           <context context-type="linenumber">226</context>
         </context-group>
       </trans-unit>
       <trans-unit id="31fedef1fc3d84d80cd88ca33aa79b2d3e7d7be7" datatype="html">
         <source>CPU requirement (cores)</source>
-        <target state="new">CPU requirement (cores)</target>
+        <target state="new">CPU请求数(核)</target>
```
Comment (hwdef) [inline i18n/zh/messages.zh.xlf]: 中英文空格。

File: i18n/zh/messages.zh.xlf [hunk 19]
```diff
@@ -3559,7 +3559,7 @@
             CPU requirement must be given as a positive number.
           </source>
         <target state="new">
-            CPU requirement 必须是正整数。
+            CPU请求核数必须是正整数。
```
Comment (hwdef) [inline i18n/zh/messages.zh.xlf]: 中英文空格。

File: i18n/zh/messages.zh.xlf [hunk 20]
```diff
@@ -3571,7 +3571,7 @@
             CPU requirement must be given as a valid number.
           </source>
         <target state="new">
-            CPU requirement 必须是有效的数字。
+            CPU请求核数必须是有效的数字。
```
Comment (hwdef) [inline i18n/zh/messages.zh.xlf]: 中英文空格。
